app/execution.js

```diff
@@ -413,7 +409,24 @@ exports.loadDataAndSendNotification = function loadDataAndSendNotification(repor
                         message: thresholdAlertEmail ? 'error while generating chart for threshold alert' + err : 'error while generating chart' + err,
                         errMsg: err,
                     });
-                    await updateSchedulerTaskLog(err, shedularlog, channelList.team);
+                    channelStatus = [];
+                    for (let index = 0; index < channels.length; index++) {
+                        channelStatus.push(
+                            {
+                                channel: channels[index],
+                                notificationSent: false,
+                                status: "Something wrong while creating an image"
```

Comment:
it does not lead to anthing. "Something wrong while creating an image". just add the correct error message